todo_app/flask_config.py

```diff
@@ -1,9 +1,10 @@
 import os
+"""dotenv.load_dot_env(".env")"""
 
 
 class Config:
     def __init__(self):
         """Base configuration variables."""
-        self.SECRET_KEY = os.environ.get('SECRET_KEY')
+        self.SECRET_KEY = 'SECRET_KEY'
```

Comment:
We don't want to leave this change permanently - as per the name the `SECRET_KEY` should be a secret so it shouldn't be hardcoded! Why have you switched to this approach?